docs/pages/kubernetes-access/discovery-apps.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Kubernetes Apps Discovery
```

Comment:
Is this the right title? The feature converts "Kubernetes Services" into "Teleport Apps." I fear introducing "Kubernetes Apps" will be confusing.

Reply:
Yep, naming was hard. First version of this documentation had "Kubernetes Services Discovery" 😅 
We also will probably have databases discovery inside Kube clusters in the future, so @tigrato suggested we call this one "Apps Discovery" and then add another section the docs side panel when we add databases, so we'll have something like

```
Kubernetes Apps Discovery
Kuberentes Databases Discovery
Kubernetes Clusters Discovery
  |- AWS
  |- Azure
  |- Google Cloud
```

Or combining Apps & Databases under "Services":
```
Kubernetes Services Discovery
|- Apps 
|- Database
Kubernetes Clusters Discovery
  |- AWS
  |- Azure
  |- Google Cloud
```

To be honest I'm not sure which way is better and open to suggestions.